evap/staff/tests/test_importers.py

```diff
@@ -459,14 +516,65 @@ def test_import_new_participant(self):
         success_messages, __, __ = PersonImporter.process_source_evaluation(
             ImportType.PARTICIPANT, self.evaluation1, test_run=True, source_evaluation=self.evaluation2
         )
-        self.assertIn("1 participants would be added to the evaluation", "".join(success_messages))
+        self.assertIn("1 participant would be added to the evaluation", "".join(success_messages))
         self.assertIn("{}".format(self.participant2.email), "".join(success_messages))
 
         success_messages, __, __ = PersonImporter.process_source_evaluation(
             ImportType.PARTICIPANT, self.evaluation1, test_run=False, source_evaluation=self.evaluation2
         )
-        self.assertIn("1 participants added to the evaluation", "".join(success_messages))
+        self.assertIn("1 participant added to the evaluation", "".join(success_messages))
         self.assertIn("{}".format(self.participant2.email), "".join(success_messages))
 
         self.assertEqual(self.evaluation1.participants.count(), 2)
         self.assertEqual(set(self.evaluation1.participants.all()), set([self.participant1, self.participant2]))
+
+    def test_person_success_messages(self):
+        success_messages, __, __ = PersonImporter.process_source_evaluation(
+            ImportType.PARTICIPANT, self.evaluation1, test_run=True, source_evaluation=self.evaluation3
```

Comment:
same with a local variable containing the arguments we don't really care about in this test.